capstone/src/components/User/Adventurer/ExperienceInfo/ExperienceInfo.jsx

```diff
@@ -33,14 +33,33 @@ function getPhotoReference(restaurant) {
   return restaurant.photos ? restaurant.photos[0].photo_reference : null;
 }
 
-export default function ExperienceInfo({ onUpdateRestaurants, indexRestaurant, onSelectRestaurant }) {
+function calculateNewMeanReviews(meanScores, reviewsNumber, newReview) {
+  return meanScores.map((scoreObject) => {
+    const newVal = (newReview.find(
+      (review) => review.feedbackId === scoreObject.feedbackId,
+    )).score;
+    return {
+      ...scoreObject,
+      meanReviewScore:
+        ((newVal / 5 * 100) + scoreObject.meanReviewScore * reviewsNumber) /
```

Comment:
Similarly here, maybe we can add a quick comment since the "5" isn't self-explanatory.

Reply:
Great! I will go ahead and fix that! 
Thanks for letting me know 😁